app/javascript/mastodon/common.js

```diff
@@ -2,7 +2,7 @@ import Rails from '@rails/ujs';
 import 'font-awesome/css/font-awesome.css';
 
 export function start() {
-  require.context('../images/', true, /\.(jpg|png|svg)$/);
+  // require.context('../images/', true, /\.(jpg|png|svg)$/);
```

Comment:
Should this line be deleted, or still WIP?

Reply:
WIP for now, I need to check if those are correctly processed by Vite in the current state (they should as `vite-rails` should add them as entrypoints).